core/logbook/src/main/java/org/phoebus/logbook/LogEntry.java

```diff
@@ -13,6 +13,10 @@
 
     public String getDescription();
 
+    public default String getSource(){
```

Comment:
Not specific to this commit/change, but `LogEntry` is API for code that uses the logbook, or code that implements the binding for a site-specific logbook.
As such, it should have javadoc to describe its methods.
This commit doesn't have to retro-actively add all the missing javadoc, but maybe describe at least the newly added method, since it's not immediately obvious:
Source of what? The body of the entry? In a certain format, or allowing plain text, html, markup, markdown, ..?